contracts/upgradeable_contracts/BaseFeeManager.sol

```diff
@@ -54,12 +54,26 @@ contract BaseFeeManager is EternalStorage, FeeTypes {
     function getFeeManagerMode() public pure returns(bytes4);
 
     function random(uint256 _count) public view returns(uint256) {
-        return uint256(blockhash(block.number.sub(1))) % _count;
+        return uint256(block.blockhash(block.number.sub(1))) % _count;
     }
 
     function distributeFeeProportionally(uint256 _fee, bytes32 _direction) internal {
         IRewardableValidators validators = rewardableValidatorContract();
-        address[] memory validatorList = validators.validatorList();
```

Comment:
This line throws the following error:
```
TypeError: Type inaccessible dynamic type is not implicitly convertible to expected type address[] memory.
```
It seems that it is possible to return variably-sized data from a method, but it just cannot be read from within the EVM before the Byzantium hardfork.
https://ethereum.stackexchange.com/questions/36637/returning-static-array-inaccessible-dynamic-type/36652

So I had to duplicate `validatorList()` logic to get the list inside `distributeFeeProportionally`. I didn't like this solution but I couldn't find any other option to make it work.